### PR TITLE
Update msmschooser.md

### PR DESCRIPTION
**Pull request by alan-jarmusch**
update to documentation with corrections, formatting changes, and the addition of a FAQ section

### CODE REVIEW

**Comment by alan-jarmusch** [comment]
@mwang87 can you please review and deploy updated documentation on MSMS-Chooser

File: docs/msmschooser.md [hunk 1]
```diff
@@ -5,38 +5,40 @@
 ## Summary
 [MSMS-Chooser](https://proteomics2.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22MS-CHOOSER%22%7D) is a GNPS workflow and open-source protocol to empower the community to collect MS/MS reference data and contribute to the public MS/MS reference library.
 
-**This is a community effort and welcome everyone to participate!**
-  * We encourage individuals to implement the protocols in their own laboratory.
-  * Individuals with chemical standards without access to instrumentation are encouraged to contact the Dorrestein lab to inquire about running standards in collaboration (see contact information below).
+**This is a community effort and welcome everyone to participate!** <br>
```
Comment (mwang87) [inline docs/msmschooser.md:8]: Shouldn't have HTML in here unless absolutely necessary

File: docs/msmschooser.md [hunk 2]
```diff
@@ -5,38 +5,40 @@
...
 ## Step-by-Step Instructions
 
 ### Completion of MSMS-Chooser Template - [Link Here](https://docs.google.com/spreadsheets/d/1C6bpcaC2b4KpXaimkpslH4whlqKlN4T9DB-BKNYshjQ/edit?usp=sharing)
-1. Make a copy of the **MSMS-Chooser Template** by selecting “File” and then “Make a copy” - continue working in Google Sheets.
+1. Make a copy of the **MSMS-Chooser Template** by selecting “File” and then “Make a copy”. We **strongly encourage** users to continue working in Google Sheets as the formatting will be maintained.
 2. Fill out the MSMS-Chooser Template in the **"MSMS-Chooser Sample Template"** tab.
 
     > Notes: <br>
-      - A readme tab is included that describes what should be entered as well as an example and additional information is in documentation (GitHub). **Important: You must record the position of each chemical in the 96-well plate.** <br>
+      - The first tab describes what should be entered as well as an example and additional information is in documentation (GitHub). **Important: You must record the position of each chemical in the 96-well plate.** <br>
```
Comment (mwang87) [inline docs/msmschooser.md:29]: HTML here too

File: docs/msmschooser.md [hunk 3]
```diff
@@ -5,38 +5,40 @@
...
+3. Download a copy of the **MSMS-Chooser Submission File** tab as a .tsv file by selecting "File", "Download", and then "Tab-separated value (.tsv, current sheet)."
 
     > Notes: <br>
       - The MSMS-Chooser File required for submission is automatically generated in the "MSMS-Chooser Submission" tab. <br>
 
-4. Download a copy of the applicable Sequence Table using the **Sequence Table Generator** tab as a .csv from Google Sheets.
-  - Please double check the chemicals and well position match in the sequence table.
+4. Download a copy of the applicable Sequence Table using the **Sequence Table Generator** tab as a .csv file from Google Sheets by selecting "File", "Download", and then "Comma-separated value (.csv, current sheet)."
+
+    > Notes: <br>
```
Comment (mwang87) [inline docs/msmschooser.md:40]: Here too

**Pull request merged**